packages/grid/_modules_/grid/hooks/features/virtualization/useGridVirtualScroller.tsx

```diff
@@ -269,8 +270,12 @@ export const useGridVirtualScroller = (props: UseGridVirtualScrollerProps) => {
 
     const rows: JSX.Element[] = [];
 
+    const isEmptySpace = nextRenderContext.lastRowIndex >= currentPage.rows.length;
```

Comment:
When `nextRenderContext.lastRowIndex` is greater than `currentPage.rows.length`? It's clamped to the number of rows.

https://github.com/mui-org/material-ui-x/blob/9e4c04f1b6d5a2ef8504c81c1cfe295c2f392ee4/packages/grid/_modules_/grid/hooks/features/virtualization/useGridVirtualScroller.tsx#L162-L166

https://github.com/mui-org/material-ui-x/blob/9e4c04f1b6d5a2ef8504c81c1cfe295c2f392ee4/packages/grid/_modules_/grid/hooks/features/virtualization/useGridVirtualScroller.tsx#L153-L156

Reply:
This is just to catch if the total number of rows is less than the number of rows that can fit in the viewport (user-visible rows). If there is a better way I'm up for it.

Reply:
Updated - I used the `scrollY` in the `GridRow`.